blocks/career-carousel/career-carousel.js

```diff
@@ -0,0 +1,158 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+import { fetchIndex } from '../../scripts/scripts.js';
+
+export function filterIncompleteEntries(json) {
+  return json.data.filter((e) => e.image !== '' && e['career-quote'] !== '0' && e['career-jobtitle'] !== '0');
```

Comment:
I also want to filter out items that do have an image but where there is either no career quote or jobtitle (these show up as `0` values because of the Excel Filter.) so I think I need to check for all of them to ensure we have all 3.